### PR TITLE
add support for onNavigationChanged

### PR DESCRIPTION
**Pull request by puneet-pdx**

### CODE REVIEW

File: toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt [hunk 1]
```diff
@@ -99,6 +99,7 @@ public fun MapView(
     backgroundGrid: BackgroundGrid = BackgroundGrid(),
     onViewpointChanged: (() -> Unit)? = null,
     onInteractingChanged: ((isInteracting: Boolean) -> Unit)? = null,
+    onNavigationChanged: ((isNavigating: Boolean) -> Unit)? = null,
```
Comment (hud10837) [inline toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt:102]: Can you add doc for this

Comment (puneet-pdx) [reply toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt:102]: updated

File: toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt [hunk 2]
```diff
@@ -208,6 +212,11 @@ private fun MapViewEventHandler(
                 }
             }
         }
+        launch(Dispatchers.Main.immediate) {
```
[the remark this answers is not in the document]
Comment (puneet-pdx) [reply toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt]: updated

**Review by hud10837 — APPROVED** [review]
👍

**Review by gunt0001 — APPROVED** [review]

**Pull request merged**